### PR TITLE
Update hello-world.md

### PR DESCRIPTION
**Pull request by reza-ebrahimi**

### CODE REVIEW

File: hello-world.md [hunk 1]
```diff
@@ -17,18 +17,13 @@ its existence. Furthermore, all language features in Rust have a predictable (an
...
-this in unsafe blocks, along with many other unsafe things. Crucially, Rust
-ensures that unsafety in unsafe blocks stays in unsafe blocks and can't affect
-the rest of your program). Finally, Rust takes many concepts from modern
-programming languages and introduces them to the systems language space.
-Hopefully, that makes programming in Rust more productive, efficient, and
-enjoyable.
+has benefits: 
+
+- Rust is memory safe.
+- Rust's type system ensures that you don't get the kind of memory errors which are common in C++. Memory leaks, accessing un-initialised memory, dangling pointers. all are impossible in Rust.
```
Comment (nrc) [inline hello-world.md:23]: This line is an explanation of memory safefy and should be kept with the above line.

File: hello-world.md [hunk 2]
```diff
@@ -17,18 +17,13 @@ its existence. Furthermore, all language features in Rust have a predictable (an
...
-the rest of your program). Finally, Rust takes many concepts from modern
-programming languages and introduces them to the systems language space.
-Hopefully, that makes programming in Rust more productive, efficient, and
-enjoyable.
+has benefits: 
+
+- Rust is memory safe.
+- Rust's type system ensures that you don't get the kind of memory errors which are common in C++. Memory leaks, accessing un-initialised memory, dangling pointers. all are impossible in Rust. 
+- All array indexing is bounds checked (of course, if you want to avoid the cost, you can (at the expense of safety)).
+- Rust allows you to do this in unsafe blocks, along with many other unsafe things. Crucially, Rust ensures that unsafety in unsafe blocks stays in unsafe blocks and can't affect the rest of your program).
```
Comment (nrc) [inline hello-world.md:25]: This line is referring to the above line, so should be together

Comment (reza-ebrahimi) [reply hello-world.md:25]: I'm just expand those for more readability.

Comment (reza-ebrahimi) [reply hello-world.md:25]: ':' is not a bullet, maybe I'm wrong. What can I should do?

**Pull request closed without merging**